src/ipsis/course-merge/BaseCollectionEntity.js

```diff
@@ -51,5 +67,14 @@ export class BaseCollectionEntity extends Entity {
 
 		return this._entity.getLinkByRel('prev').href;
 	}
+
+	clearAllEntities() {
+		this._allEntities = null;
+	}
```

Comment:
Do we need this anymore? https://github.com/Brightspace/ipsis-sis-course-merge/pull/65#discussion_r1092203882

Reply:
Done